### PR TITLE
Post Cantina Audit Minor Changes

### PR DESCRIPTION
**Pull request by oldchili**

### CODE REVIEW

File: src/VoteDelegate.sol [hunk 1]
```diff
@@ -102,7 +102,7 @@ contract VoteDelegate {
     }
 
     function reserveHatch() external {
-        require(block.number > hatchTrigger + HATCH_SIZE + HATCH_COOLDOWN, "VoteDelegate/cooldown-not-finished");
+        require(block.number >= hatchTrigger + HATCH_SIZE + HATCH_COOLDOWN, "VoteDelegate/cooldown-not-finished");
```
Comment (sunbreak1211) [inline src/VoteDelegate.sol:105]: So are we finally changing this?

File: src/VoteDelegateFactory.sol [hunk 2]
```diff
@@ -59,7 +59,7 @@ contract VoteDelegateFactory {
     }
 
     function create() external returns (address voteDelegate) {
-        voteDelegate = address(new VoteDelegate{salt: keccak256(abi.encode(msg.sender))}(chief, polling, msg.sender));
```
Comment (sunbreak1211) [inline src/VoteDelegateFactory.sol:62]: Hmm shouldn't we just avoid the `msg.sender` as also proposed before?

Comment (oldchili) [reply src/VoteDelegateFactory.sol:62]: Their final recommendation was to do it this way.
While both would probably work I think it makes sense for the msg.sender to be the salt, acting as a unique identifier for the vote-delegate.
Gas is less of an issue here (vote-delegate creations are pretty rare, and the diff should be minor anyway).

File: src/VoteDelegate.sol (hunk 1 above)
Comment (oldchili) [reply src/VoteDelegate.sol:105]: Don't see a big reason to insist here.
They are right that the triggering block could be viewed as still part of the cooldown period (as you can still lock in it and the blocking is only on the next block).
Looking at it that way it makes sense to use `>=`.

The 5/25 ratio thing is more of a nicety/bonus here.

File: src/VoteDelegateFactory.sol (hunk 2 above)
Comment (sunbreak1211) [reply src/VoteDelegateFactory.sol:62]: Hmmm, not really sure why the value of the salt should really matter, but if feeling more comfortable leaving it then fine. I was a bit surprised they left only that option as recommendation.

**Review by telome — APPROVED** [review]
I'd also tend to use empty salt but fine to leave as-is as well. LGTM.

**Review by sunbreak1211 — APPROVED** [review]
LGTM

**Pull request merged**